controllers/application_controller.go

```diff
@@ -179,15 +187,12 @@ func (r *ApplicationReconciler) objectStatuses(ctx context.Context, resources []
 		}
 		s, err := status(resource)
 		if err != nil {
-			// Just logging the error for now. Not sure if this is the right way to handle it.
-			logger.Error(err, "unable to compute status for resource", "gvk", resource.GroupVersionKind().String(),
```

Comment:
Is logging not needed ?

Reply:
Also there is a reason why we continue and not bail out. 
A single bad object would cause us from not aggregating status from other objects. It was a bad UX.

Reply:
Will add logging back and continue compute status from other resources even if error occurs.
Will do in a separate PR to keep this one clean.